samples/nrf9160/modem_shell/Kconfig

```diff
@@ -209,6 +209,13 @@ config MOSH_LWM2M_ENDPOINT_PREFIX
 	help
 	  Sets the prefix of the endpoint name used when connecting to the LwM2M server.
 
+# Make AVsystem the default LwM2M server.
```

Comment:
Why would you move it out of the overlay?

Reply:
Currently, AVSystem Coiote is the only LwM2M server the location services work with. So, it makes sense to have it as the default. And it's always possible to override the default configuration if necessary.